### DISCUSSION TITLE
When creating a new difficulty/map, some default settings should be changed

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#24371] [Ideas]
Enable countdown should be Off by default since not many new maps actually use that, so it being off is a standard
Widescreen support should be On by default since widescreen monitors are a standard at the moment

**Comment by peppy** [comment]
I'm so confused. Did you check before making this post?

![2023-07-26 23 56 07@2x](https://github.com/ppy/osu/assets/191335/2a3c511c-b905-4ab8-9536-8f4ffae49be8)

**Reply by Skubers** [reply]
Ok so widescreen support is checked by default that's my bad, but Countdown is also checked by default for me

![image](https://github.com/ppy/osu/assets/82095491/b9b1adfa-3eb4-43b1-a4a8-58c260a00963)

**Reply by peppy** [reply]
wasn't for me, so no idea.

**Reply by peppy** [reply]
i may have had a local change affecting this
